PostPairsTrading.md

```diff
@@ -0,0 +1,247 @@
+
+![LinearRegression](resources/linear_regression.png)
+
+The plotted graph above illustrates the relationship between the NASDAQ 100 and the S&P 500 indices, with each purple dot representing a data point of their prices at a given time. The linear trend visible in the scatter plot suggests a strong positive cointegration between the two indices. By applying linear regression, we can model this relationship mathematically, allowing us to predict the NASDAQ 100 index price based on the S&P 500 index price. This predictive power is crucial for pair trading, as it helps identify mispricings and potential trading opportunities.
+
+Linear regression aims to identify relationships between historical data, which we then extrapolate to current data. The differences between these relationships, or deviations, are our spreads. We've already calculated the 𝛼 and 𝛽 using the logarithmic values of our historical data (since real-time price values for priceX and priceY are unknown). Now, we simply combine everything and apply linear regression to our price logarithms:
+
+$$spread = log(priceY) - (\beta \cdot log(priceX)+\alpha)$$
+
+```q
+q)spreads: log[priceY] - alpha + log[priceX] * beta
+-0.1493929 0.0451223 -0.08835117 0.0451223 0.1579725
+```
+
+There are different methods we can use to obtain the best alpha and beta values that minimize the spreads or, in other words, there are mathematical methods to find the line that best fits the prices.
+
+The aim of this post is not to delve deeply into these methods but to mention that the most popular method is called the least squares method. For this case, it provides a closed-form solution that depends on our historical data. This means we do not need any iterative algorithm or a more complex method to find these optimal alpha and beta values.
+
+>💡 For those interested in our implementation of these formulas in kdb+/q, the code can be found in our repository [Pair-Trading](https://github.com/hablapps/pairstrading/blob/5-Post/linear_regression.q).
+
+This precisely meets our objective—a **comprehensive method for representing relative changes between both assets**. As we can deduce, our mean is now 0 because our assets are normalized, cointegrated and on the same scale. Therefore, ideally, the differential between their prices should be 0. Consequently, when our spread is below 0, we infer that asset X is overpriced, whereas if it's above 0, then asset Y is overpriced.
+
+
+## Real-Time Pair Party
+
+Now that we have selected a pair of cointegrated indices and understand how to calculate their relationships, let's see how we can create a real-time pair trading scenario.
+
+> ⚠️ An important note is that this post will include a real-time simulation. In other words, if we wanted to develop a 100% real-time product, we would need to make slight adjustments to the code.

```

Comment:
Does this post include a real-time simulation? I think we aren't discussing that further on, I mean, we aren't showing data here. Maybe you just wanted to refer to the repo.

Reply:
Could we move this warning further on? Later, you mention again that this is a simulation ("In our case, we are simulating real time; we do not have a 100% real-time product.") and I feel it's redundant. Could we integrate both references in the same paragraph?